### ISSUE TITLE
RadListView itemDeselected: args.view is undefined (iOS only)

### ISSUE THREAD

**Issue by NickIliev** [telerik/nativescript-ui-feedback#411]
`args.view` is undefined for iOS on `itemDeselected` event (works as expected on Android)

Steps to reproduce:
Use [this example](https://github.com/telerik/nativescript-ui-samples/blob/release/sdk/app/listview/selection-states/selection-states-model.ts#L22-L30) and log the `args.view`

```
public itemSelected(args: listViewModule.ListViewEventData) {
    console.log("itemSelected args.view: " + args.view);
    var item = this.dataItems.getItem(args.index);
    item.selected = true;
}

public itemDeselected(args: listViewModule.ListViewEventData) {
    console.log("itemDeselected args.view: " + args.view); // undefined for iOS
    var item = this.dataItems.getItem(args.index);
    item.selected = false;
}
```

reported via t.1141137

**Comment by zbranzov** [comment]
Still valid with 3.6.1

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-listview@3.8.0.
